instagram/src/components/PostContainer/PostContainer.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import CommentSection from "../CommentSection/Comments";
+import PropTypes from "prop-types";
+import "./Post.css";
+
+const PostContainer = props => {
+  return (
+    <div className="post-wrapper">
```

Comment:
Definitely could convert some of these blocks to components to make this page less verbose